test/globals-node.js

```diff
@@ -0,0 +1,25 @@
+/*jshint node:true */
+(function() {
+
+QUnit.module( "globals for Node.js only" );
+
+QUnit.test( "QUnit exports", function( assert ) {
+	var qunit = require( "../dist/qunit" );
+
+	assert.ok( qunit, "Required module QUnit truthy" );
```

Comment:
I removed one of the following assertions but I intend to keep this one simple assertion.